pages/animals.tsx

```diff
@@ -5,11 +5,12 @@ import { CenteredMain } from '../components/Centered';
 import { checkAnswer } from '../services/checkAnswer';
 import Image from 'next/image';
 import { useInterval } from '../services/useInterval';
+import getRandomNumber from '../services/getRandomNumber';
```

Comment:
This file and some other files changed because I moved some code into a shared helper, and then all those files I touched got autofromatted by prettier. Prettier is still formatting in a weird way that I don't like, so I'll try to fix that soon.